JebraAzureFunctions/JebraAzureFunctions/tools.cs

```diff
@@ -336,7 +336,7 @@ public static QuestionModel SimplifySquareRoots()
             QuestionModel questionModel = new QuestionModel();
             questionModel.answer_a = num.ToString();
             questionModel.answer_b = (0 - num).ToString();
-            questionModel.question = "sqrt(" + square + ")";
+            questionModel.question = "$" + "sqrt{" + square + "}" + "$";
```

Comment:
The `\sqrt` function needs a backslash (`\`) before it. Since C# interprets backslashes as escape characters, you need to escape the backslash by using a double backslash here (`\\`), as such:

```suggestion
            questionModel.question = "$\\sqrt{" + square + "}$";
```